alpine-rails-nginx/etc/nginx/nginx.conf

```diff
@@ -21,9 +16,9 @@ http {
   tcp_nopush  on;
   tcp_nodelay on;
 
-  keepalive_timeout 90;
+  keepalive_timeout 300;
```

Comment:
k, I'm fine with that. Was just curious if its from other projects, or something particular you noticed on this PR.